### PR TITLE
feat: edit/delete/reply for emoji/sticker/audio

### PR DESCRIPTION
**Pull request by OmarBasem**
This is a pull request that completes the functionalities needed for issue [#13246 on status-mobile](https://github.com/status-im/status-mobile/pull/13808).

Add missing actions (edit/delete/reply) to certain types of messages (emoji/sticker/audio)
To implement the needed functionalities there are 2 pull requests to this repo `status-go` and to `status-mobile`.

The pull request on status-mobile includes a full description of the features and how it is implemented.

### CODE REVIEW

File: protocol/requests/edit_message.go
Comment (Samyoul) [inline protocol/requests/edit_message.go:5]: This will give you a linting error with our CI. Putting the line back will appease the great delinter.

**Review by Samyoul — APPROVED** [review]
Nice PR. Clean work.

**Review by richard-ramos — APPROVED** [review]

**Comment by churik** [comment]
@Samyoul how better to manage it, as it contains 5 commits from forked repo? 
maybe to create a copy of this PR directly into our repo and merge it?

I see both PRs on the status-react and status-go side are a bit stuck due to the limited possibility of rebasing the forked repo.

@OmarBasem sorry for delay!

**Comment by Samyoul** [comment]
Thanks @churik, I don't see a problem with merging this PR.

@OmarBasem are you able to merge this PR? If not I am able to merge this for you.

**Comment by OmarBasem** [comment]
@Samyoul No, I can't merge it as I do not have write access.

**Comment by churik** [comment]
@Samyoul can you please rebase it, squash the commits, I'll check for regression and then we merge?

**Pull request merged**
